classes_assignment.js

```diff
@@ -0,0 +1,61 @@
+// class Human {
+//   constructor(name, age){
+//     this.name = name;
+//     this.age = age;
+//   }
+// ageOneYear() {
+//   return this.age++
+// }
+//
+// eating(str) {
+//   return "mmm, mmm, mmm, I\'m love\'n it"
+// }
```

Comment:
When a question ask for you to log the result, it means you should console.log within that function.